lib/benefits/compute-javascript.ts

```diff
@@ -164,6 +165,14 @@ export const CONDITION_STATEGY: ConditionsLayout = {
       return false
     },
   },
+  invert: {
+    test: (condition, props) => {
+      return !CONDITION_STATEGY[condition.value.type].test(
+        condition.value,
+        props
+      )
```

Comment:
Il me semble que ce serait bien de créer une catégorie a part (par exemple `modifiers`) parce que ce invert n'a pas la même fonction que les autres entrées de l'objet CONDITION_STRATEGY (ce n'est pas un test, c'est une fonction qui modifie un test existant).

Cela crée une confusion a 2 endroits :

* au moment d'appeler la fonction avec cet objet nesté qui contient des clefs similaires mais qui n'ont pas la même signification 
```javascript
{
    type: "invert",
    value: { // ici "value" contient à la fois le test à appeler ET la valeur à passer audit test
         type: "epcis", // (le fait que epci soit ultra polysémique au sein de l'application n'aide pas a deviner que c'est le nom d'un test)
         values: [excludedEPCI], // ici value ne contient que la valeur à passer au test
    },
},
```
* ça crée aussi une confusion ici où il faut faire attention à bien savoir de quel `type` et `value` on parle

si on met ça a part dans un objet (par exemple) `STRATEGY_MODIFIERS` on peut se permettre d'avoir un appel plus lisible 

```javascript
{
  modifier: 'invert',
  testParams : {
    type: "epci",
    values: [excludedEpci]
}

// dans compute-javascript
  invert: {
    test: (condition, props) => {
      return !CONDITION_STATEGY[condition.testParams.type].test(
        condition.testParams,
        props
      )
    },

Reply:
mais peut-être pour un second temps parce que ça demande de faire un tri entre les modifiers et les tests

Reply:
Je crois que je vois ce que tu veux dire. Je le formulerais différemment, en effet, pour moi, il faudrait distinguer des opérateurs finaux (les existants) et des opérateurs combinatoires (`not`, `and`, `or`).